jsinfo/calcExtend.js

```diff
@@ -0,0 +1,47 @@
+//old junked code, please ignore
+
+//function calculate(str) {
```

Comment:
make sure to clean up any commented code that is either not useful anymore or doesn't serve any purpose in the current PR.